mender-client.sh

```diff
@@ -162,15 +201,29 @@ function get_deplyoment() {
 # $1 - deployment id
 # $2 - enum (installing, downloading, rebooting, success, failure, already-installed)
 function set_deplyoment_status() {
-  echo "${1}: state changed to: ${2}"
-  curl -k \
+  [[ -z "$1" || "$1" == "" ]] && { log "ERROR: \$1 (deployment id) is empty"; exit 1; }
+  [[ -z "$2" || "$2" == "" ]] && { log "ERROR: \$2 (status) is empty"; exit 1; }
```

Comment:
The test here is redundant, it is enough with:
```suggestion
  [[ -z "$1" ]] && { log "ERROR: \$1 (deployment id) is empty"; exit 1; }
  [[ -z "$2" ]] && { log "ERROR: \$2 (status) is empty"; exit 1; }
```